### PR TITLE
[ci] rework spread and bors interaction

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

**Comment by Saviq** [comment]
bors try

File: bors.toml [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
+status = [
+          "BuildAndTest (lxd:alpine-3.15:spread/build/alpine:amd64)",
+          "BuildAndTest (google:ubuntu-20.04-64:spread/build/sbuild:debian_sid)",
+          "BuildAndTest (google:ubuntu-20.04-64:spread/build/sbuild:ubuntu)",
+          "BuildAndTest (google:ubuntu-20.04-64:spread/build/sbuild:ubuntu_impish)",
+          "BuildAndTest (google:ubuntu-20.04-64:spread/build/sbuild:ubuntu_arm64)",
+          "BuildAndTest (google:ubuntu-20.04-64:spread/build/ubuntu:rpi)",
+          "BuildAndTest (google:ubuntu-20.04-64:spread/build/ubuntu:clang)",
+          "BuildAndTest (google:fedora-34-64:spread/build/fedora:amd64)",
+          "BuildAndTest (google:fedora-35-64:spread/build/fedora:amd64)",
```
Comment (AlanGriffiths) [inline bors.toml:10]: AIUI these will be checked before Bors merges? In which case should `debian_sid` and `ubuntu:rpi` be in the list?

Comment (Saviq) [reply bors.toml]: Indeed.

**Review by AlanGriffiths — APPROVED** [review]
I hope this is an improvement...

bors r+

**Pull request merged**